legate/driver/ui.py

```diff
@@ -0,0 +1,246 @@
+# Copyright AS2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Helpler functions for simple text UI output.
+
+The color functions in this module require ``colorama`` to be installed in
+order to generate color output. If ``colorama`` is not available, plain
+text output (i.e. without ANSI color codes) will generated.
```

Comment:
will be

Reply:
https://github.com/nv-legate/legate.core/pull/371/commits/696f620de288d4cf4b442359409c55d298518013